bigquery/samples/tests/test_routine_samples.py

```diff
@@ -55,3 +64,39 @@ def test_create_routine_ddl(capsys, client, random_routine_id):
         )
     ]
     assert routine.arguments == expected_arguments
+
+
+def test_list_routines(capsys, client, dataset_id, routine_id):
```

Comment:
What's the reason for recombining the tests in this file?

Reply:
I was thinking it will be a good point to combine tests in one file. If it is causing any problems or misunderstandings I can split them back.

Reply:
I know that in the beginning, they were represented in one file. Then I separated them in #9212, but now I think it was a bad idea. I started to look at them in the same way as `model `or `dataset_label` operations.

Reply:
OK. Makes sense to recombine.